docs/website/site.eleventy.cjs

```diff
@@ -32,6 +32,54 @@ module.exports = function(eleventyConfig) {
 
   eleventyConfig.addFilter('search', searchFilter);
 
+  eleventyConfig.addFilter('replaceString', (value) => {
```

Comment:
is this the best name? i'm guessing all "addLiquidFilter" replace things in strings ;)

Reply:
plus,  i don't see it used anywhere :D

Reply:
yeah, right, this one I left by chance, it was an old one :)
And about liquid filters... I was trying to write it in a better way, but they were not working with more than one parameter.